pyqtgraph/opengl/items/GLVolumeItem.py

```diff
@@ -140,10 +139,14 @@ def paint(self):
         mat_mvp = self.mvpMatrix()
         mat_mvp = np.array(mat_mvp.data(), dtype=np.float32)
 
-        view = self.view()
+        # calculate camera coordinates in this model's local space.
+        # (in eye space, the camera is at the origin)
+        modelview = self.modelViewMatrix()
+        cam_local = modelview.inverted()[0].map(QtGui.QVector3D())
```

Comment:
Do you know if we get the same rounding errors that we get w/ `QTransform.inverted()` ?

Reply:
Looking a little more closely, maybe it's ok?

https://github.com/qt/qtbase/blob/0770171dcda24a567d6f57b87fdd284011e56e16/src/gui/math3d/qmatrix4x4.cpp#L316-L390